diff --git a/contrib/board/Config.in.linux_arg b/contrib/board/Config.in.linux_arg
index c29954d..ca91105 100644
--- a/contrib/board/Config.in.linux_arg
+++ b/contrib/board/Config.in.linux_arg
@@ -10,3 +10,4 @@ source "contrib/board/corewind/core9g25/Config.in.linux_arg"
 source "contrib/board/axentia/sama5d3_linea/Config.in.linux_arg"
 source "contrib/board/mini-box/pico_sam9g45/Config.in.linux_arg"
 source "contrib/board/stephan/sama5d3_stephan/Config.in.linux_arg"
+source "contrib/board/laird/wb50n/Config.in.linux_arg"
